mods/cnc/maps/desert-rats-cnc/map.yaml

```diff
@@ -161,7 +161,7 @@ Actors:
 	Actor41: t08
 		Owner: Neutral
 		Location: 23,39
-	Actor42: split2
+	Actor42: T03.Transformable
```

Comment:
Desert Rats is a MP map - was this intentional?

Reply:
Depending on how we want to have it here?

Reply:
IMO best to leave MP maps as they are for now, and then let mappers design/update maps explicitly to use these.